apps/microtvm/reference-vm/zephyr/Vagrantfile

```diff
@@ -18,6 +18,18 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "tlcpack/microtvm-zephyr-2.5"
 
+  if ENV.has_key?("RVM_NUM_CORES")
```

Comment:
maybe we want to prefix all env vars with TVM_?

Reply:
Done.